package-lock.json

```diff
@@ -0,0 +1,13297 @@
+{
```

Comment:
The project uses yarn as the default package manager. Usually it's not recommended to have a package-lock.json and a yarn.lock json in the same project. I probably should add this to the readme.

Reply:
sure let's yank those